contracts/exchange-forwarder/contracts/src/MixinExchangeWrapper.sol

```diff
@@ -88,7 +88,7 @@ contract MixinExchangeWrapper is
         )
     {
         // No taker fee or percentage fee
-        if (order.takerFee == 0 || order.takerFeeAssetData.equals(order.makerAssetData)) {
+        if (order.takerFee == 0 || _isPercentageFee(order.takerFeeAssetData, order.makerAssetData)) {
```

Comment:
It seems unlikely but possible that one of the taker `assetDatas` are using the `ERC20BridgeProxy`, in which case we should use this function for the `if (order.takerFeeAssetData.equals(order.takerAssetData))` check too.

Reply:
I thought it was impossible for the `takerAssetData` or `takerFeeAssetData` to use the bridge proxy? The bridges require a balance of the opposing asset when performing a `bridgeTransferFrom`, but since the Forwarder contract is the taker, it will be the one that holds a `makerAsset` balance when the `ERC20BridgeProxy` is dispatched

Reply:
This is only true if using the DEX aggregation pattern where the bridge is also the maker. However, other bridge contracts can require some sort of allowances and pull from the user's account (which would be true of margin limit orders).

Reply:
ah yeah, I forgot about that side of things